### ISSUE TITLE
Big bunch of records

### ISSUE THREAD

**Issue by debjeetdas** [akveo/ng2-smart-table#503]
I've a large chunk of data and I don't want to load the whole info at once.
Is there any workaround for server side pagination.??

**Comment by AntonGrekov** [comment]
Yes, sure, look for ServerDataSource. It's well documented on home page of library.
You can make it work with server side filtration pretty easy.
